events/wc_frostwolf_events.txt

```diff
@@ -922,62 +984,65 @@ province_event = {
 		d_frostwolf = {
 			holder_scope = {
 				if = {
-					limit = { frostwolf_exodus_d_frostwolf_trigger = yes }
+					limit = { frostwolf_leader_exists = yes }
 					narrative_event = { id = WCFW.21 }
```

Comment:
Still trying to get to the south, but some kind of evil mage always teleports me to Lordaeron.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/67303194-2373a800-f503-11e9-89c1-333593e05386.png)

</details>

Reply:
Thanks for scrubbing through the chain, should be resolved with latest commit.

Reply:
Actually, this part still needs looking at. Void my earlier statement.

Reply:
Resolved in latest commit (`WCFW.21` to `WCFW.41`) as can be seen here:
```
narrative_event = { id = WCFW.41 }
```